src/Wallabag/CoreBundle/Entity/Entry.php

```diff
@@ -244,6 +244,15 @@ class Entry
      * )
      */
     private $tags;
+    
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="origin_url", type="text", nullable=true)
+     *
```

Comment:
`nullable=true` is there not an inconsistency with the migration script ?

Reply:
The migration is now: `'notnull' => false,`